build/package/circleci/orb.yml

```diff
@@ -51,7 +51,6 @@ commands:
               -repoType=${LD_REPO_TYPE} \
               -repoUrl=${LD_REPO_URL} \
               -repoName=${CIRCLE_PROJECT_REPONAME} \
-              -repoHead=${CIRCLE_BRANCH} \
```

Comment:
we can assume that in CLI situations people will manually have the desired branch checked out, and hopefully github/bitbucket pre-checkout the correct branch, but does circle provide the repo with the correct branch checked out?

Reply:
Yes, circle checks out to the correct branch.